src/app/store/chains/stx-chain.selectors.ts

```diff
@@ -1,18 +1,12 @@
-import { useAsync } from 'react-async-hook';
-import { useSelector } from 'react-redux';
-
 import { Wallet } from '@stacks/wallet-sdk';
 import memoize from 'promise-memoize';
 
 import { InternalMethods } from '@shared/message-types';
 import { RequestDerivedStxAccounts } from '@shared/messages';
 
 import { RootState } from '@app/store';
-import { useCurrentKeyDetails } from '@app/store/keys/key.selectors';
-
-import { useDefaultWalletSecretKey } from '../in-memory-key/in-memory-key.selectors';
 
-const selectStxChain = (state: RootState) => state.chains.stx;
+export const selectStxChain = (state: RootState) => state.chains.stx;
```

Comment:
I still think blockchains should be spelled out. The chain really isn't `stx`, or am I wrong here?